README.md

```diff
@@ -46,7 +47,33 @@ docker pull ghcr.io/jooola/earhorn
 ```
 Usage: earhorn [OPTIONS]
 
-  See the ffmpeg documentation for details about the `--archive-segment-*` options:
+  ENVIRONMENT VARIABLES:
+
+  If a `.env` file is present in the current directory, it will be loaded and can be used to pass environment
+  variables to this tool.
+
+  ARCHIVE STORAGE:
+
+  The storage can be defined using a path to a local directory or an url to an s3 bucket. Segments will be saved on
+  the storage you specified.
+
+  To use an s3 bucket, you need to install the `s3` extras (`pip install earhorn[s3]`), use `s3://bucket-name` as
+  value for the `--archive-path` option and export the s3 bucket credentials listed in the table below:
+
+  | Variable                | Description                               | Example                     |
+  | ----------------------- | ----------------------------------------- | --------------------------- |
+  | AWS_S3_ENDPOINT_URL     | The endpoint to your s3 bucket (optional) | https://s3.nl-ams.scw.cloud |
+  | AWS_ACCESS_KEY_ID       | The access key for your bucket user       | AKIA568knmklmk              |
+  | AWS_SECRET_ACCESS_KEY   | The secret key for your bucket user       | mi0y84wu498zxsasa           |
+  | AWS_STORAGE_BUCKET_NAME | The name of the bucket to upload to       | earhorn-bucket              |
+  | AWS_S3_REGION_NAME      | Region of your s3 bucket                  | us-east-2                   |
```

Comment:
I am unsure about how those variables are handled, does `AWS_STORAGE_BUCKET_NAME` override any value we provide in the `s3://bucket-name` option ?

Could we remove most of those variables from the earhorn code and let boto3 handle it ? Or is it required to be passed via function calls ?

Which of those variables are official and which are ours ?